### PR TITLE
Add link to migration tool for Trello

### PR DESCRIPTION
**Pull request by maxammann**
* Resolves: None
* Target version: master 

### Summary

I developed this tool quite a white back and it is used actively by users who used trello and want to migrate to deck (including myself). I think it makes sense to link to it as Deck claims to be "similar to Trello" in the description of this Github project.

This also may lead to new users who are currently using Trello.

### TODO

None

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [x] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by stefan-niedermann** [comment]
Maybe some section like "Known 3rd party clients" makes sense to keep track of API users (which then also can be notified properly for example when breaking changes are planned or so), @juliushaertl ?

**Comment by maxammann** [comment]
Works for me too :) just wanted to do minimum changes. If we integrate a seapate section then it is even better.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Deck is a kanban style organization tool aimed at personal planning and project
 - Discuss with your team using comments
 - Keep track of changes in the activity stream
 - Get your project organized
+- [Migrate](https://github.com/maxammann/trello-to-deck) from Trello
```
Comment (juliushaertl) [inline README.md:17]: I'm happy to add this to the readme, though we should have a dedicated section for 3rd party integrations like the mobile apps section as this list is the general feature list of the app itself.

File: README.md [hunk 2]
```diff
@@ -15,11 +15,16 @@ Deck is a kanban style organization tool aimed at personal planning and project
...
 ### Mobile apps
 
-- The [Nextcloud Deck app for Android](https://github.com/stefan-niedermann/nextcloud-deck) is available in the [Google Play Store](https://play.google.com/store/apps/details?id=it.niedermann.nextcloud.deck.play) 
+- [Nextcloud Deck app for Android](https://github.com/stefan-niedermann/nextcloud-deck) - It is available in [F-Droid](https://f-droid.org/de/packages/it.niedermann.nextcloud.deck/) and the [Google Play Store](https://play.google.com/store/apps/details?id=it.niedermann.nextcloud.deck.play)
 
-![Deck - Manage cards on your board](http://download.bitgrid.net/nextcloud/deck/screenshots/1.0/Deck-2.png)
+### 3rd-Party Integrations
+
+- [trello-to-deck](https://github.com/maxammann/trello-to-deck) - Migrates cards from Trello
+- [mail2deck](https://github.com/newroco/mail2deck) - Provides an "email in" solution
```
Comment (maxammann) [inline README.md:27]: I added this one as requested by some other user: https://github.com/maxammann/trello-to-deck/issues/23#issuecomment-748645153

File: README.md [hunk 3]
```diff
@@ -15,11 +15,16 @@ Deck is a kanban style organization tool aimed at personal planning and project
 - Keep track of changes in the activity stream
 - Get your project organized
 
+![Deck - Manage cards on your board](http://download.bitgrid.net/nextcloud/deck/screenshots/1.0/Deck-2.png)
```
Comment (maxammann) [inline README.md:18]: I moved this screenshot up, as it is more important than the addons below

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**